grails-app/controllers/org/bbop/apollo/GroupController.groovy

```diff
@@ -74,14 +77,14 @@ class GroupController {
             // restricted groups
             def groups = dataObject.groupId ? [UserGroup.findById(dataObject.groupId)] : UserGroup.all
             def filteredGroups =  groups
+            def currentUser = User.findByUsername(dataObject.username)
             // if user is admin, then include all
             // if group has metadata with the creator or no metadata then include
-
-            if (!permissionService.isAdmin()) {
```

Comment:
This is is an identical all to isUSerAdmin, unless there is a bug in it.   Either is fine.